ehrapy/preprocessing/_scanpy_pp_api.py

```diff
@@ -225,6 +232,23 @@ def neighbors(
         method: Use 'umap' [McInnes18]_ or 'gauss' (Gauss kernel following [Coifman05]_ with adaptive width [Haghverdi16]_) for computing connectivities.
                 Use 'rapids' for the RAPIDS implementation of UMAP (experimental, GPU only).
         metric: A known metric’s name or a callable that returns a distance.
+        transformer
+            Approximate kNN search implementation following the API of
+            :class:`~sklearn.neighbors.KNeighborsTransformer`.
+            See :doc:`/how-to/knn-transformers` for more details.
```

Comment:
The link breaks in our documentation as this was from the scanpy doc; but we could insert the full scanpy doc link instead https://scanpy.readthedocs.io/en/latest/how-to/knn-transformers.html & make a quick comment that this is scanpy but works the same for ehrapy